src/ahbicht/expressions/requirement_constraint_expression_evaluation.py

```diff
@@ -195,10 +150,10 @@ def _then_also(
         """
         if other_condition.conditions_fulfilled != ConditionFulfilledValue.NEUTRAL:
             evaluated_composition = EvaluatedComposition(conditions_fulfilled=other_condition.conditions_fulfilled)
-            format_constraint_is_required = other_condition.conditions_fulfilled.value
+            format_constraint_is_required = other_condition.conditions_fulfilled == ConditionFulfilledValue.FULFILLED
```

Comment:
Wieso ändert sich das `.value` hier zu `== ConditionFulfilledValue.FULFILLED` ?

Reply:
Ah, um wieder ein boolean daraus zu machen, richtig?

Reply:
> Ah, um wieder ein boolean daraus zu machen, richtig?

ja. vorher (im **a**enum) war der value in zwei von vier fällen ein boolean